pkg/base/testdata/raft_config

```diff
@@ -2,19 +2,19 @@ echo
 ----
 (base.RaftConfig) {
  RaftTickInterval: (time.Duration) 200ms,
- RaftElectionTimeoutTicks: (int) 15,
+ RaftElectionTimeoutTicks: (int) 10,
  RaftHeartbeatIntervalTicks: (int) 5,
- RangeLeaseRaftElectionTimeoutMultiplier: (float64) 3,
+ RangeLeaseRaftElectionTimeoutMultiplier: (float64) 2.5,
  RangeLeaseRenewalFraction: (float64) 0.5,
  RaftLogTruncationThreshold: (int64) 16777216,
  RaftProposalQuota: (int64) 8388608,
  RaftMaxUncommittedEntriesSize: (uint64) 16777216,
  RaftMaxSizePerMsg: (uint64) 32768,
  RaftMaxCommittedSizePerReady: (uint64) 67108864,
  RaftMaxInflightMsgs: (int) 128,
- RaftDelaySplitToSuppressSnapshotTicks: (int) 245
+ RaftDelaySplitToSuppressSnapshotTicks: (int) 230
 }
-RaftElectionTimeout: 3s
-RangeLeaseDurations: active=9s renewal=4.5s
-NodeLivenessDurations: active=9s renewal=4.5s
-SentinelGossipTTL: 4.5s
+RaftElectionTimeout: 2s
+RangeLeaseDurations: active=5s renewal=2.5s
+NodeLivenessDurations: active=5s renewal=2.5s
+SentinelGossipTTL: 2.5s
```

Comment:
Could this be too tight? I think this needs to propagate through the entire gossip network within that timeout if things are to work completely (I think the absence of the sentinel triggers new gossip connections).

Reply:
Yeah, I need to audit all the dependant parameters throughout the code and make sure it's all reasonable. I just wanted to get some input on the basic tweaks first.

Reply:
Changed this from the lease renewal interval to the lease expiration interval, i.e. from 4.5s to 5.0s. It seems more natural to to use the same gossip and lease expirations anyway.